src/backend/dx11/src/lib.rs

```diff
@@ -1053,7 +1054,9 @@ impl hal::Device<Backend> for Device {
         &self,
         surface: &mut Surface,
         config: hal::SwapchainConfig,
-    ) -> (Swapchain, hal::Backbuffer<Backend>) {
+        old_swapchain: Option<Swapchain>,
+        _physical_device: &PhysicalDevice,
```

Comment:
I agree with @msiglreith that it doesn't seem right to accept a `PhysicalDevice` in a `Device` method.